executor/index_lookup_join_test.go

```diff
@@ -81,7 +81,8 @@ func (s *testSuite1) TestIndexJoinUnionScan(c *C) {
 		"2 2 2 2 2",
 		"2 2 4 2 4",
 	))
-	tk.MustQuery("select /*+ INL_MERGE_JOIN(t1, t2)*/ * from t1 join t2 on t1.a = t2.a").Check(testkit.Rows(
+	// INL_MERGE_JOIN is invalid
```

Comment:
Why the test need to Sort()? ...... Hash table will influences IMJ？ Or the hint is invalid and use HashJoin to do this?

Reply:
> Why the test need to Sort()? ...... Hash table will influences IMJ？ Or the hint is invalid and use HashJoin to do this?

t1.a does not have indexes, so this query does not use INL_MJ in fact.